### PR TITLE
Long diagnostic for E0538

### PR DESCRIPTION
**Pull request by Havvy**
r? @GuillaumeGomez

### CODE REVIEW

File: src/libsyntax/diagnostic_list.rs [hunk 1]
```diff
@@ -93,6 +93,36 @@ For more information about the cfg attribute, read:
 https://doc.rust-lang.org/reference.html#conditional-compilation
 "##,
 
+E0538: r##"
+Attribute contains multiple of the same meta item.
```
Comment (GuillaumeGomez) [inline src/libsyntax/diagnostic_list.rs]: What about "Attribute contains same meta item more than once"? (it's a real question, I'm not an english native so it's just a suggestion!)

**Comment by GuillaumeGomez** [comment]
I'd like to have an english native's opinion on my suggestion, but otherwise, it looks good for me. Good job! :)

File: src/libsyntax/diagnostic_list.rs (hunk 1 above)
Comment (Havvy) [reply src/libsyntax/diagnostic_list.rs]: That is slightly better wording.

**Comment by GuillaumeGomez** [comment]
Then let's go for it!

@bors: r+ rollup

**Pull request merged**